tests/singlyLinkedList.test.ts

```diff
@@ -0,0 +1,73 @@
+import { SinglyLinkedList } from '../src/singlyLinkedList';
+
+describe('SinglyLinkedList Functionality: ', () => {
+	const newList = new SinglyLinkedList();
+
+	describe('new instance', () => {
+		it('has null head/tail', () => {
+			expect(newList.get(0)).toBeNull();
+		});
+	});
+
+	describe('.insertLast(value) & .push(value) method', () => {
+		it('adds node to end of list', () => {
+			newList.insertLast('1');
+			newList.push({ hello: 'world' });
+			expect(newList.get(0)).toEqual('1');
+			expect(newList.get(1)).toEqual({ hello: 'world' });
+		});
+	});
+
+	describe('.getLength', () => {
+		it('return the length of the list', () => {
+			expect(newList.getLength()).toEqual(2);
```

Comment:
Might suggest a `beforeEach` method as a cleanup to isolate all the tests from each other, making them more deterministic. (This test case depends on a previous test case)

Reply:
Good call. Made that change :-D